scripts/common/rook.sh

```diff
@@ -173,3 +173,238 @@ function maybe_cleanup_rook() {
 function rook_osd_phase_ready() {
     [ "$(kubectl -n rook-ceph get cephcluster rook-ceph --template '{{.status.phase}}')" = 'Ready' ]
 }
+
+function current_rook_version() {
```

Comment:
one day we are gonna have a name collision and its not gonna be good...

Reply:
yeah `rook_version` is already claimed...